setup.py

```diff
@@ -37,7 +37,7 @@ def get_package_data(root_dir):
 install_requires = [
     'apscheduler', 'attrs', 'dulwich', 'IPython', 'jupyter', 'lmfit', 'matplotlib',
     'numdifftools', 'numpy>=1.23', 'opencv-python', 'pandas', 'PyQt5', 'pyqtgraph<0.13.0', 'pyvisa', 'pyzmqrpc', 'qcodes>=0.30.0',
```

Comment:
You can unpin pyqtgrapgh. It is not required any more, but it does interfere with other packages (which have pyqtgraph>=0.13)

Reply:
I unpinned pyqtgraph!